x-pack/plugins/index_management/public/store/reducers/row_status.js

```diff
@@ -26,6 +27,14 @@ import {
 } from '../../../common/constants';
 
 export const rowStatus = handleActions({
+  [clearRowStatus](state, action) {
```

Comment:
I edited this handler to immediately return `state` and bypass the rest of the logic here, and the UX was the same: I tried to delete an index while logged in as a user with insufficient permissions, got a danger toast, and the status remained unchanged in the table. Is there another way to test this logic?